protocol-designer/src/components/modals/EditModulesModal/index.tsx

```diff
@@ -282,73 +296,117 @@ const EditModulesModalComponent = (
     return filteredOptions
   }
 
+  const [field] = useField('selectedSlot')
+
   return (
-    <Modal
-      heading={i18n.t(`modules.module_long_names.${moduleType}`)}
-      className={cx(modalStyles.modal, styles.edit_module_modal)}
-      contentsClassName={styles.modal_contents}
-    >
-      <>
-        {slotIssue && (
-          <PDAlert
-            alertType="warning"
-            title={i18n.t('alert.module_placement.SLOT_OCCUPIED.title')}
-            description={''}
-          />
-        )}
-        <Form>
-          <div className={styles.form_row}>
-            <FormGroup label="Model" className={styles.option_model}>
-              <ModelDropdown
-                fieldName="selectedModel"
-                tabIndex={0}
-                options={getModuleOptionsForRobotType(
-                  MODELS_FOR_MODULE_TYPE[moduleType],
-                  robotType
-                )}
-              />
-            </FormGroup>
-            {showSlotOption && (
-              <>
-                {!enableSlotSelection && (
-                  <Tooltip {...tooltipProps}>{slotOptionTooltip}</Tooltip>
-                )}
-
-                <div {...targetProps} className={styles.option_slot}>
-                  <FormGroup label="Position">
-                    <SlotDropdown
-                      fieldName="selectedSlot"
-                      options={getAllModuleSlotsByType(moduleType, robotType)}
-                      disabled={!enableSlotSelection}
-                      tabIndex={1}
-                    />
-                  </FormGroup>
-                </div>
-
-                <ConnectedSlotMap
-                  fieldName="selectedSlot"
-                  robotType={robotType}
+    <ModalShell width="48rem">
+      <Box marginTop={SPACING.spacing32} paddingX={SPACING.spacing32}>
```

Comment:
If the following works

```suggestion
    <ModalShell width="48rem" paddingTop={SPACING.spacing32}>
      <Box paddingX={SPACING.spacing32}>
```

Recently I read this article.
https://mxstbr.com/thoughts/margin/